redux-form/lib/reduxForm.d.ts

```diff
@@ -50,14 +41,14 @@ export interface Config<FormData extends DataShape, P, S> {
     asyncBlurFields?: string[];
 
     /**
-     * a function that takes all the form values, the dispatch function,
-     * the props given to your component and the current blurred field,
-     * and returns a Promise that will resolve if the validation is passed,
-     * or will reject with an object of validation errors in the form { field1: <String>, field2: <String> }.
+     * a function that takes all the form values, the dispatch function, and
+     * the props given to your component, and returns a Promise that will
+     * resolve if the validation is passed, or will reject with an object of
+     * validation errors in the form { field1: <String>, field2: <String> }.
      *
      * See Asynchronous Blur Validation Example for more details.
      */
-    asyncValidate?(values: FormData, dispatch: Dispatch<S>, props: P, blurredField: string): Promise<any>;
+    asyncValidate?(values: FormData, dispatch: Dispatch<S>, props: P): Promise<any>;
```

Comment:
`blurredField` is still passed https://github.com/erikras/redux-form/blob/master/src/reduxForm.js#L375